### PR TITLE
Add stacks

### PR DESCRIPTION
**Pull request by johnsudaar**

### CODE REVIEW

File: _scalingo_api/stacks.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+--- row ---
+
+**Stack attributes**
+
+{:.table}
+| field       | type    | description                             |
+| ---         | ---     | ---                                     |
+| id          | string  | unique ID, starts with 'st-'            |
+| name        | string  | stack name display name                 |
```
Comment (EtienneM) [inline _scalingo_api/stacks.md]:
```suggestion
| name        | string  | stack display name                 |
```

File: _scalingo_api/stacks.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+||| col |||
+
+```json
+{
+  "id": "st-44b8ae3c-db5d-4e17-9344-65dcbfab095e",
+  "name": "scalingo-18",
+  "base_image": "scalingo/builder-18:v7",
+  "default": true,
+  "created_at": "2019-04-15T16:15:00.147Z",
+  "description": "Runtime stack based on Ubuntu 18:04"
```
Comment (EtienneM) [inline _scalingo_api/stacks.md]:
```suggestion
  "description": "Runtime stack based on Ubuntu 18.04"
```

And there are other places you made the same typo

Comment (johnsudaar) [reply _scalingo_api/stacks.md]: This typo was in prod data. Fixed in the 2 regions and in the article

Comment (EtienneM) [reply _scalingo_api/stacks.md]: OK great, thanks :)

**Pull request merged**